src/multibody/geometry.hpp

```diff
@@ -274,8 +287,15 @@ namespace se3
     
     ///
     /// \brief Check if at least one of the collision pairs has its two collision objects in collision.
+    ///        The results are stored in the vector GeometryData::collision_results.
+    ///
+    /// \param[out] the index of the colliding pair if the function returns true.
     ///
-    bool isColliding() const;
+    bool isColliding(Index& collidingPair);
```

Comment:
I dont like too much to have a redundancy of methods doing very closed stuff. Could we pass the collidingPair results by side effect in the GeoemtryData structure? And then only keep the second method, with explicit description of the side effect in the doc?